Resources/public/templates/tabs/relations.hbt

```diff
@@ -36,6 +37,9 @@
             {{/each}}
         </tbody>
         </table>
+        {{else}}
+            <p>This content has no related content</p>
```

Comment:
this deserves to be styled a bit, I would say it should be in italic and you should also removed the default margin.